skin-resque-frontend/src/pages/ColorPallete/Info.js

```diff
@@ -0,0 +1,25 @@
+import { Link } from 'react-router-dom';
+import { Button, Heading, Text } from "../../components"
+import './Info.scss'
+
+function Info() {
+  return (
+	<div className='page' style={{ backgroundImage: `url('${process.env.PUBLIC_URL}/images/bg-cosmetics.svg')` }}>
```

Comment:
Background doesn't load on first view of the page. Besides that `/images/...` should be enough

Reply:
I think it should be in scss file as well

Reply:
> Background doesn't load on first view of the page. Besides that `/images/...` should be enough

`/images/...` isnt enough, that was the only way It worked for me

Reply:
> I think it should be in scss file as well

Can you show me how to do it? This was the only way it was working for me

Reply:
![image](https://user-images.githubusercontent.com/83178883/209876995-a64b0d8a-56f3-41b4-82d0-47c15962b11d.png)